crates/anvil/core/src/eth/mod.rs

```diff
@@ -321,7 +321,10 @@ pub enum EthRequest {
     TraceBlock(BlockNumber),
 
     // Return filtered traces over blocks
-    #[cfg_attr(feature = "serde", serde(rename = "trace_filter",))]
+    #[cfg_attr(
+        feature = "serde",
+        serde(rename = "trace_filter", with = "single_or_1_item_sequence")
```

Comment:
isn't this just?

`with = "sequence"`

Reply:
Yes if we're fine to break the single object path. :+1: